server/src/main/java/keywhiz/Environments.java

```diff
@@ -0,0 +1,22 @@
+package keywhiz;
+
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import keywhiz.KeywhizConfig;
+
+public final class Environments {
+  private Environments() {}
+
+  public static Environment fromBootstrap(Bootstrap<KeywhizConfig> bootstrap) {
+    Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+
+    return new Environment(
+        bootstrap.getApplication().getName(),
+        bootstrap.getObjectMapper().copy(),
+        validator,
+        bootstrap.getMetricRegistry(),
+        bootstrap.getClassLoader());
+  }
+}
```

Comment:
In DropWizard, it appears that non-server commands have access to Bootstrap but not the full Environment. However, it's easy to construct the latter from the former, as our test runner demonstrates. This just repackages that logic into non-test code for use by commands so that they can make use of the same Injector as the service itself.